src/flamegraph/attrs.rs

```diff
@@ -202,15 +209,14 @@ mod test {
         let r = s.as_bytes();
 
         let mut expected_inner = HashMap::new();
-        let foo_attrs: IndexMap<String, String> = indexmap! {
-            "class".to_owned() => "foo class".to_owned(),
-            "xlink:href".to_owned() => "foo href".to_owned(),
-            "target".to_owned() => "foo target".to_owned(),
-            "gextra1".to_owned() => "gextra1".to_owned(),
-            "gextra2".to_owned() => "foo gextra2".to_owned(),
-            "aextra1".to_owned() => "foo aextra1".to_owned(),
-            "aextra2".to_owned() => "foo aextra2".to_owned(),
-        };
+        let mut foo_attrs = FnvHashMap::default();
+        foo_attrs.insert("class".to_owned(), "foo class".to_owned());
+        foo_attrs.insert("xlink:href".to_owned(), "foo href".to_owned());
+        foo_attrs.insert("target".to_owned(), "foo target".to_owned());
+        foo_attrs.insert("gextra1".to_owned(), "gextra1".to_owned());
+        foo_attrs.insert("gextra2".to_owned(), "foo gextra2".to_owned());
+        foo_attrs.insert("aextra1".to_owned(), "foo aextra1".to_owned());
+        foo_attrs.insert("aextra2".to_owned(), "foo aextra2".to_owned());
```

Comment:
I'd probably add a `dev-dependencies` on `maplit` to clean this up :)

Reply:
Unfortunately it doesn't work with `FnvHashMap`.

Reply:
`.into_iter().collect()` :joy: